utils/crawlPlayersInfo.js

```diff
@@ -0,0 +1,113 @@
+const puppeteer = require("puppeteer");
+const { KBO_PLAYER_SEARCH_URL } = require("../constants/crawling");
+
+const crawlPlayersInfo = async (players) => {
+  const newPlayers = [];
+  for (let i = 0; i < players.length; i += 1) {
+    newPlayers.push({ ...players[i] });
+  }
+
+  const browser = await puppeteer.launch();
+
+  const pages = [];
+  for (let i = 0; i < players.length; i += 1) {
+    pages.push(browser.newPage());
+  }
+
+  const createdPages = await Promise.all(pages);
+
+  const searchPages = [];
+  for (let i = 0; i < players.length; i += 1) {
+    const page = createdPages[i];
+    searchPages.push(page.goto(KBO_PLAYER_SEARCH_URL + players[i].name));
+  }
+
+  await Promise.all(searchPages);
+
+  const links = [];
+  for (let i = 0; i < players.length; i += 1) {
+    const searchCompletePage = createdPages[i];
+    const currentPlayer = players[i];
+
+    links.push(searchCompletePage.evaluate((toBeMatchedPlayer) => {
+      const $rows = document.querySelectorAll(".tEx tBody > tr");
+
+      for (let j = 0; j < $rows.length; j += 1) {
+        const $row = $rows[j];
+
+        const backNumber = $row.children[0].textContent.trim();
+        const name = $row.children[1].textContent.trim();
+        const team = $row.children[2].textContent.trim();
+
+        const isMatched = (
+          backNumber !== "#"
+          && toBeMatchedPlayer.name === name
+          && toBeMatchedPlayer.team === team
```

Comment:
```suggestion
          backNumber !== "#"
            && toBeMatchedPlayer.name === name
            && toBeMatchedPlayer.team === team
```
삼항 연산자는 인덴팅을 주는데 다른 operator는 어떤지 잘 모르겠습니다.. 이따 얘기해봐요~~